radio/src/gui/common/stdlcd/popups.cpp

```diff
@@ -94,7 +94,19 @@ const char * runPopupMenu(event_t event)
     drawVerticalScrollbar(MENU_X+MENU_W-1, y+1, MENU_MAX_DISPLAY_LINES * (FH+1), popupMenuOffset, popupMenuItemsCount, display_count);
   }
 
-  switch (event) {
+  event_t eventTemp = event;
+
+#if defined(ROTARY_ENCODER_NAVIGATION) && !defined(COLORLCD)
+  if (g_eeGeneral.rotEncDirection >= 2) {
```

Comment:
Please make defines for the different rotary encoder modes and ause then at all places where rotEncDirection is checked.

Reply:
Done as enums.